composer.json

```diff
@@ -9,7 +9,7 @@
     },
     "minimum-stability": "stable",
     "require": {
-        "php": ">=5.4,<8.0-dev",
+        "php": "~5.6.0",
```

Comment:
If the tests pass for PHP 7.0 I still wouldn't want to recommend running it in production without significant manual testing.
So let's drop PHP 7 testing for now.